src/lib.rs

```diff
@@ -1,77 +1,128 @@
-//! # `Monotonic` implementation based on DWT and SysTick
+//! # `Monotonic` implementation based on DWT cycle counter and SysTick
 
 #![no_std]
 
 use cortex_m::peripheral::{syst::SystClkSource, DCB, DWT, SYST};
-pub use fugit::{self, ExtU32};
+pub use fugit::{self, ExtU32, ExtU64};
```

Comment:
Maybe `ExtU64` could be conditionally imported, based on `extend` feature?

Reply:
How would that help or what problem would be solved by doing so?

Reply:
Nothing major, maybe even nitpick tier, but I don't think there's any dependency on `ExtU64` when `extend` is false?

```suggestion
#[cfg(feature = "extend")]
pub use fugit::ExtU64;
pub use fugit::{self, ExtU32};
```

Reply:
Sorry missed this, the correct `ExtXXX` for each version is needed.

Else it will break glob imports.

Reply:
To clarify, reexport `ExtU32` when not using `extended` and reexport `ExtU64` when using `extended`.

Reply:
My take would be that's only four additional lines of code to review and maintain for no added value. I can't see how re-exporting a trait at that point in the namespace that is not used in some feature configurations could cause any problem.
But changed as requested.